Samples/WinMLSamplesGallery/WinMLSamplesGallery/SampleData/SampleData.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
can we discuss the design here? It's not obvious to me why this data lives in a separate json file and I'm curious what the reasons are.

Reply:
Discussed in standup.